modules/ROOT/pages/errors/gql-errors/42N29.adoc

```diff
@@ -0,0 +1,4 @@
+= 42N29
+
+== Status description
+error: syntax error or access rule violation - unbound variables in pattern expression. Pattern expressions are not allowed to introduce new variables: `{ $var }`.
```

Comment:
```suggestion
error: syntax error or access rule violation - unbound variables in pattern expression. Pattern expressions are not allowed to introduce new variables: `{ $variable }`.
```

This was already wrong before the split, but I didn't notice it until now. If we decide to keep the status descriptions in the index file as well, it should be updated in both places.